include/effects.h

```diff
@@ -142,6 +142,8 @@ enum EffectID {
     EFFECT_86                       = 0x86,
 };
 
+typedef struct EffectInstance EffectInstance;
```

Comment:
wut dis do

Reply:
`LightningFXData` has references to `EffectInstance`, so we need to declare it beforehand.

Reply:
I think preferably we do `struct EffectInstance*` in LightningFXData